src/demos/multi_file_viewer_demo.vue

```diff
@@ -0,0 +1,172 @@
+<template>
+  <div class="multi-file-viewer-demo">
+
+    <div>
+      <p @click="open_file(1)"> Add A </p>
+      <p @click="open_file(2)"> Add B </p>
+      <p @click="open_file(3)"> Add C </p>
+      <p @click="open_file(4)"> Add D </p>
+      <p @click="open_file(5)"> Add E </p>
+      <div class="mvf-container">
+        <multi-file-viewer
```

Comment:
Can you add some text (visible in when viewing the demo) explaining the difference between what these two multi-file-viewers are demonstrating, if any?